### ISSUE TITLE
"--tail-color-fadeout" param issue

### ISSUE THREAD

**Issue by szolnokit** [gpx-animator/gpx-animator#387]
1.7.0 java CLI working as expected... until add:
`--tail-color-fadeout false `

Gets:
```
Exception in thread "main" java.lang.AssertionError
        at app.gpx_animator.ui.cli.CommandLineConfigurationFactory.<init>(CommandLineConfigurationFactory.java:201)
        at app.gpx_animator.Main.start(Main.java:55)
        at app.gpx_animator.Main.main(Main.java:44)

```

**Comment by therealrobster** [comment]
I can verify this is happening also.  I can make it work via the GUI, but if I use it in the terminal, I get the same error:

	Exception in thread "main" java.lang.AssertionError
		at app.gpx_animator.ui.cli.CommandLineConfigurationFactory.<init>(CommandLineConfigurationFactory.java:201)
		at app.gpx_animator.Main.start(Main.java:55)
		at app.gpx_animator.Main.main(Main.java:44)

For now I will have to use no fade out.
I trust the above helps.

**Comment by McPringle** [comment]
Fixed by @ky0n with PR #476 - thank you very much for your work!
